nupic/research/frameworks/dendrites/modules/dendritic_layers.py

```diff
@@ -86,16 +93,25 @@ def forward(self, x, context):
 
 
 class BiasingDendriticLayer(DendriticLayerBase):
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.dendritic_bias = DendriticBias()
```

Comment:
The layer class (`BiasingDendriticLayer`) now has an attribute which is another class, in this case `DendriticBias`. If I understand correctly, the purpose of this "inner class" is to subclass `torch.nn.Module` and define `forward` so that hooks can be added (for metrics purposes), is that right?

Reply:
That's correct, I'd say.

Small note, technically `BiasingDendriticLayer` doesn't have an attribute which is another class. It has an attribute which is an _instance_ of another class `torch.nn.Module` (if it was a class, as opposed to instance, it'd be `self.dendritic_bias = DendriticBias`. 

Semantics aside, everything else is spot on.